### PR TITLE
Add support for zstd-compressed archives (Python version)

### PR DESCRIPTION
**Pull request by bastianeicher**

### CODE REVIEW

File: zeroinstall/zerostore/unpack.py [hunk 1]
```diff
@@ -131,6 +132,11 @@ def check_type_ok(mime_type):
 			raise SafeException(_("This package looks like a xz-compressed package, but you don't have the 'unxz' command "
 					"I need to extract it. Install the package containing it (it's probably called 'xz-utils') "
 					"first."))
+	elif mime_type == 'application/x-zstd-compressed-tar':
+		if not find_in_path('unzstd'):
+			raise SafeException(_("This package looks like a zstd-compressed package, but you don't have the 'unzstd' command "
+					"I need to extract it. Install the package containing it (it's probably called 'zstd') "
+					"first."))
```
Comment (talex5) [inline zeroinstall/zerostore/unpack.py:139]: As with the OCaml version, I think this should be (`pass	# We can get it through Zero Install`). The case above it looks wrong too.

Comment (bastianeicher) [reply zeroinstall/zerostore/unpack.py]: Changed, like in the OCaml version, thanks. :)

**Comment by bastianeicher** [comment]
@talex5 Could you publish a new version of the 2.3 branch with this feature?

**Comment by talex5** [comment]
Do you need me to do it? I was trying to remember how the new release process works, but it looks like you made the last release anyway! https://github.com/0install/0install/releases/tag/v2.3.15

**Comment by bastianeicher** [comment]
Indeed! I've published [v2.3.16](https://github.com/0install/0install/releases/tag/v2.3.16) now.

**Pull request merged**